cmd/bootstrap/cmd/finalize_test.go

```diff
@@ -53,6 +49,29 @@ const finalizeHappyPathLogs = "^deterministic bootstrapping random seed" +
 
 var finalizeHappyPathRegex = regexp.MustCompile(finalizeHappyPathLogs)
 
+// getFirstQCSignerPath picks first internal node as participant who will do the QC signing
+// node needs to have random beacon key and node info to be treated as internal.
+func getFirstQCSignerPath(t *testing.T, bootDir string) string {
```

Comment:
This is not necessary, because we do not need any actual private keys to produce a QC from votes.